fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -91,12 +106,11 @@ fun TabItem(
             tint = iconColor
         )
     }
-
     if (textAlignment == TabTextAlignment.HORIZONTAL) {
         ConstraintLayout(
             modifier = modifier
                 .then(clickableModifier)
-                .background(backgroundColor)
```

Comment:
Why is this removed?

Reply:
to remove the background shadow that appears when clicked or pressed